bench/lib/Experiments.hs

```diff
@@ -328,27 +350,35 @@ runBench runSess Bench {..} = handleAny (\e -> print e >> return badRun)
       changeDoc doc [hygienicEdit]
       waitForProgressDone
 
-
     liftIO $ output $ "Running " <> name <> " benchmark"
     (runSetup, userState) <- duration $ benchSetup doc
-    let loop 0 = return True
-        loop n = do
+    let loop userWaits delayedWork 0 = return $ Just (userWaits, delayedWork)
```

Comment:
I love a good space leak :) - from a cursor read, it looks like the first two variables will be space leaks, but can't be sure - best to just shove a ! in front of all three.

Reply:
good catch!

Reply:
Although hopefully the GHC strictness analyser would figure that out as well. I wonder if that would make a good hover provider.